### PR TITLE
Clean TS Regression fix

### PR DESCRIPTION
**Pull request by HoussameH**
Regression missing data handle and warnings

### CODE REVIEW

File: code/lib/regression.py [hunk 1]
```diff
@@ -186,13 +192,19 @@ def clean_TS_regression(paths, param, tech):
...
+                TS_regression[region] = GenTS["TS_Max"].values
+                nanval = nanval + region + ", "
 
     # Save TS_regression as CSV
     TS_regression.to_csv(paths[tech]["TS_regression"], sep=";", decimal=",", index=True)
     create_json(
         paths[tech]["TS_regression"], param, ["author", "comment", tech, "region_name", "subregions_name", "year"], paths, ["FLH_regression", tech]
     )
+    if nanval != "":
+        warn("Missing FLH for regions: " + nanval.rstrip(", ") + "\nTime series have not been scalled for these regions")
```
Comment (kais-siala) [inline code/lib/regression.py]: scaled

File: code/config.py [hunk 2]
```diff
@@ -107,12 +107,12 @@ def scope_paths_and_parameters(paths, param):
...
-    paths["spatial_scope"] = PathTemp + "gadm36_GHA_0.shp"
-    paths["subregions"] = PathTemp + "gadm36_GHA_0.shp"
+    paths["spatial_scope"] = PathTemp + "gadm36_ECU_0.shp"
+    paths["subregions"] = PathTemp + "gadm36_ECU_0.shp"
 
     # Name tags for the scope and the subregions
-    param["region_name"] = "Ghana"  # Name tag of the spatial scope
-    param["subregions_name"] = "gadm36_GHA_0"  # Name tag of the subregions
+    param["region_name"] = "Ecuador"  # Name tag of the spatial scope
+    param["subregions_name"] = "gadm36_ECU_0"  # Name tag of the subregions
```
Comment (kais-siala) [inline code/config.py]: I disapprove of the name tags for subregions. Please see how I named them in the folders inside "Ecuador"

File: code/config.py [hunk 3]
```diff
@@ -63,9 +63,9 @@ def general_settings():
     fs = os.path.sep
     current_folder = os.path.dirname(os.path.abspath(__file__))
     # For personal Computer:
-    root = str(Path(current_folder).parent.parent.parent) + fs + "Database_KS" + fs
```
Comment (kais-siala) [inline code/config.py]: these changes are specific to your computer, should not be in Master

File: code/lib/physical_models.py [hunk 4]
```diff
@@ -450,7 +450,7 @@ def calc_CF_wind(hour, reg_ind, turbine, m, n, merraData, rasterData):
 
     # Load MERRA data, increase its resolution, and fit it to the extent
     w50m_h = resizem(merraData["W50M"][:, :, hour], m, n)
-    w50m_h = w50m_h[reg_ind]
+    w50m_h = w50m_h[tuple(reg_ind)]
```
Comment (kais-siala) [inline code/lib/physical_models.py]: why?

Comment (HoussameH) [reply code/lib/physical_models.py]: Using a list of arrays instead of a tuple will become depreciated in the future.
Raises a future_warning

**Pull request closed without merging**